action.yml

```diff
@@ -208,4 +208,13 @@ runs:
             cd ${SITE_ROOT}
           fi
 
+          echo "Deleting Pantheon PR multidev environment..."
           terminus build:env:delete:pr $PANTHEON_SITE --yes
+
+          echo "Finding closed PRs to clean up GitHub environments..."
+          CLOSED_PRS=$(gh pr list --state closed --json number --jq '.[].number')
+          for PR_NUM in "${CLOSED_PRS}"; do
+            ENV_NAME="pr-$PR_NUM"
```

Comment:
this cleanup is relying on the convention some environments being named by `pr-NUM`. I'm concerned this will miss some environments that should be deleted.

Reply:
We should not be concerned about all multidev environments, only those that are created by the action itself. Is there a mechanism within the action by which users can customize the naming convention of branches? If so, we should reuse that logic, otherwise, if branches will always be named `pr-` from this action, then we should not touch other environments that are named differently.

Reply:
As I've just added support for manually triggered workflows that don't use the `pr-` naming convention, I might be walking back this statement. But we should still be able to pull the specific environment out of the inputs. We don't need a more robust cleanup for ad hoc, non `pr-` deployments, I don't think, if we're cleaning up when the PR is merged.

Reply:
Maybe the recommendation for manual workflows is that a cleanup step is something that is manually added to that workflow that we document. We have no way of knowing what the implementation would look like for naming conventions outside of the `pr-` branches we create.